test/tornado_tests/test_motor_web.py

```diff
@@ -169,17 +169,16 @@ def test_content_type(self):
             ('foo.jpg', 'jpeg'),
             ('foo.png', 'png'),
             ('ht.html', 'html'),
-            ('jscr.js', 'javascript'),
         ]:
             # 'fs' is PyMongo's blocking GridFS
             self.fs.put(b'', filename=filename)
             for method in 'GET', 'HEAD':
                 response = self.fetch('/' + filename, method=method)
                 self.assertEqual(200, response.code)
                 # mimetypes are platform-defined, be fuzzy
-                self.assertTrue(
-                    response.headers['Content-Type'].lower().endswith(
-                        expected_type))
+                self.assertIn(
+                    expected_type,
+                    response.headers['Content-Type'].lower())
```

Comment:
This is a copy/paste from test_aiohttp_gridfs.py to give a better error message when the assertion fails.